content/en/docs/ambient/usage/ztunnel/index.md

```diff
@@ -94,17 +90,17 @@ caption="Basic ztunnel L4-only datapath"
 
 The figure depicts ambient pod workloads running on two nodes W1 and W2 of a Kubernetes cluster. There is a single instance of the ztunnel proxy on each node. In this scenario, application client pods C1, C2 and C3 need to access a service provided by pod S1 and there is no requirement for advanced L7 features such as L7 traffic routing or L7 traffic management so no Waypoint proxy is needed.
```

Comment:
I would suggest deleting the entire `## Data Plane Overview` section out of the "usage" guide, and putting the contents into your new `architecture/traffic-routing/index.md` doc (and linking to that from the usage guide)

Reply:
(agreed - but this needs to be done with the creating of an "About ztunnel" concept page, and then a "Using ztunnel" page.  this PR only touches this page because I needed to change some links.